doc/continuous_integration/codecov.md

```diff
@@ -0,0 +1,18 @@
+### Codecov
+
+Codecov is a service used to aggregate and monitor code coverage results, rendering test statistics generated by CI pipelines into interactive analytics, improving the visibility of the project's health and providing feedback for potential contributions. More info on Codecov can be found here:
+
+* [Codecov](https://codecov.io/)
+* [Navigation2 on Codecov](https://codecov.io/gh/ros-planning/navigation2)
+
+Codecov is configured 
+
+* [About the Codecov yaml](https://docs.codecov.io/docs/codecov-yaml)
+* [codecov.yml Reference](https://docs.codecov.io/docs/codecovyml-reference)
+* [codecov.yml](/codecov.yml)
+
+A custom script within the repo is reused to collect and post process the generated code coverage results from debug test jobs. This scrip simply invokes lcov on the overlay workspace to output `full_coverage.info`, and then filter this down to `workspace_coverage.info` by removing and irrelevant subdirectories, e.g. for message or test packages.
```

Comment:
reused? Wouldn't this be "used"?

"script" spelling

"This scrip simply invokes lcov on the overlay workspace **after testing is completed** ..."